scripts/update_android_ios_dependencies.py

```diff
@@ -516,7 +596,7 @@ def replace_module_line(m):
 # Regex to match lines like:
 # implementation 'com.google.firebase:firebase-auth:1.2.3'
```

Comment:
It will be good to update the example here.

Reply:
Done!